client/my-sites/stats/stats-upsell-modal/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Gridicon } from '@automattic/components';
+import { Button, Modal } from '@wordpress/components';
+import { close } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+export default function StatsUpsellModal( {
+	onClose,
+	onSubmit,
+}: {
+	onClose: () => void;
+	onSubmit: () => void;
+} ) {
+	const translate = useTranslate();
+	return (
+		<Modal
+			className="stats-upsell-modal"
+			onRequestClose={ onClose }
+			shouldCloseOnClickOutside={ false }
+			__experimentalHideHeader={ true }
```

Comment:
Needed to use this in order to match the design / avoid this header section:

![Screenshot 2023-12-19 at 14-39-06 Jetpack Stats ‹ Site Title — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/1a88bfeb-9d1a-484c-abe4-ce56987cfcf3)